omnigibson/simulator.py

```diff
@@ -99,7 +99,22 @@ def _launch_app():
     if gpu_id is not None:
         config_kwargs["active_gpu"] = gpu_id
         config_kwargs["physics_gpu"] = gpu_id
+
+    # Omni's logging is super annoying and overly verbose, so suppress it by modifying the logging levels
+    if not gm.DEBUG:
+        import sys
+        from numba.core.errors import NumbaPerformanceWarning
+        import warnings
+        sys.argv.append("--/log/level=warning")
+        sys.argv.append("--/log/fileLogLevel=warning")
+        sys.argv.append("--/log/outputStreamLevel=error")
+        warnings.simplefilter("ignore", category=NumbaPerformanceWarning)
+
+    import omni.log
+    omni_log = omni.log.get_log()
+    omni_log.enabled = False
```

Comment:
What does this do, does it disable the whole launch feed? Do we want to do that? Also should this also follow the debug mode stuff?

Reply:
It only prunes the annoying initial logging at the beginning of launch, not the whole thing